src/v/kafka/client/transport.h

```diff
@@ -14,7 +14,11 @@
 #include "kafka/protocol/api_versions.h"
 #include "kafka/protocol/create_topics.h"
 #include "kafka/protocol/fetch.h"
+#include "kafka/protocol/find_coordinator.h"
```

Comment:
this description is phenomenal. can we use it in the cover letter as well

Reply:
you betcha